spec/feature/post_show_spec.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.describe 'Hello world', type: :feature do
```

Comment:
## [OPTIONAL]
- [ ] Kindly add meaningful descriptions to these tests. kindly Remove the `hello wold`.